week-1/J-functions/exercise.js

```diff
@@ -1,7 +1,8 @@
 function halve(number) {
-  // complete the function here
+  // recomplete the function here
+  return number;
```

Comment:
This function is meant to halve the number you passed in. So if you call it like this - `halve(6)`, it's meant to give you 3.
Here, you're returning the same number, so if you call it `halve(6)` it will give you 6.
How can you fix that?

Reply:
@martynamaron I am sorry I just noticed that but it was in my brain .I changed it .I have divided number by 2 inside function and returned it .now I am getting the half of the value (3).